api/src/main/java/jakarta/data/repository/Sort.java

```diff
@@ -91,46 +98,23 @@ public boolean ignoreCase() {
         return ignoreCase;
     }
 
+    // Override to provide method documentation:
     /**
+     * Indicates whether to sort the property in ascending order (true) or descending order (false).
+     *
      * @return Returns whether sorting for this property shall be ascending.
      */
     public boolean isAscending() {
-        return Direction.ASC.equals(direction);
+        return isAscending;
```

Comment:
We can remove this as well.

Reply:
We will get the same implementation from record if we remove it, but the documentation will be lost.  I have a comment just above the method clarifying why it is there, `Override to provide method documentation:`